src/plugincode/format_csv.py

```diff
@@ -0,0 +1,232 @@
+#
```

Comment:
This file and all the format plugins (not the format plugin specs) should be in `formattedcode` .